WebApp/src/actions/timetableActions.js

```diff
@@ -215,8 +215,21 @@ export function exchangeConfirm(blockTo) {
         dispatch(action);       
         dispatch(loadExchangeRequests());
       })
-      .catch(() => {
-        window.alert("Pri vytváraní žiadosti nastala chyba.");
+      .catch((error) => {        
+        if (error.response) {
+          window.alert("Pri vytváraní žiadosti nastala chyba.");
+          // The request was made and the server responded with a status code
+          // that falls out of the range of 2xx
+        } else if (error.request) {
+          window.alert("Nepodarilo sa nadviazať spojenie so serverom.");
+          // The request was made but no response was received
+          // `error.request` is an instance of XMLHttpRequest in the browser and an instance of
+          // http.ClientRequest in node.js
+        } else {
+          // Something happened in setting up the request that triggered an Error
+          console.log('Error', error.message);          
```

Comment:
Console should not be enabled because this code goes to production and user shouldn't find out on the console what an error is. Better to use some type of logger if something like that is in React and pop window that says sorry something went bad.

Reply:
Thank you for your suggestion, i haven't noticed this happens during switching lessons with 2 accounts, switching works fine but pop-up window with error will show up again. But it's different error as before, there is actual error response from server. I will look into it and check whats back-end sendind to front-end. Also i will remove console output.